cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt

```diff
@@ -225,7 +225,6 @@ script "js_get_anomalies", type: "javascript" do
 	    "Instance Type":"instance_type"
 	    "Region":"region"
 	    "Operation":"operation"
-      "Operating System":"operating_system"
```

Comment:
Have you tested this updated policy to make sure it is now behaving as expected? Give it a go in the F1 Demo tenant

Reply:
Looks to be working, operating system option is no longer available for selection and the results are as expected.